tests/mssqltestutils.py

```
@@ -122,6 +118,10 @@ def create_test_db():
                     raise AssertionError("Database creation failed: retry logic for SQL " \
```

Can we surface the status error message in the failure here as well?